src/affirmations.js

```diff
@@ -112,6 +112,36 @@ const affirmations = [
     source: "unknown",
     author: "rupi kaur"
   }
```

Comment:
Thanks @unikornintech for the contribution. It looks like you are missing a comma on line 114.